button-merchant/src/main/java/com/usebutton/merchant/Order.java

```diff
@@ -25,14 +25,17 @@
 
 package com.usebutton.merchant;
 
+import android.content.Context;
 import android.support.annotation.Nullable;
 
 import java.util.Date;
 import java.util.List;
 import java.util.Map;
 
 /**
- * Represents an order placed by the user to be tracked using ButtonMerchant.trackOrder(order).
+ * Represents an order placed by the user to be tracked using
+ * {@link ButtonMerchant#trackOrder(Context, Order, UserActivityListener)} and
+ * {@link ButtonMerchant#reportOrder(Context, Order, OrderListener)}
  */
 public class Order {
```

Comment:
Methods with javadocs that have the `@Deprecated` annotation need a pairing `@deprecated` **javadoc annotation**. Otherwise they will not be marked deprecated when we generate the docs.